### ISSUE TITLE
Extra Frenzy Billing

### ISSUE THREAD

**Issue by sharpaypal** [sharpaypal/financial-services-support#32]
## Task For Matt 

**Give a TLDR of what the task is.**

Frenzy is looking to bill merchants an additional amount for more advanced Frenzy features. 

**Is this something Matt can do himself, or will it require other stakeholders?**

Other stakeholders (Nick, Louise) 

**How long will the task take to complete? Is there a known due date?**

Not known. 

**What are the steps that will need to be taken, to complete this?**
Is there anyone or any team that should be involved/notfied?

I'm meeting with Kevin Donnelly on Sept 19th (owning the project) to get more information. Once we have that meeting and get some clarification, we'll have an idea of how to proceed. 

**Anything else to include?** 

n/a 

@sharpaypal

**Comment by sharpaypal** [comment]
Met with Kevin last week to gather more context and after speaking with billing devs, determined editing the subscription or promo code will not be an option. Spoke with Kevin, and this is what was provided:

**Best way to bill for this:**
I broke down some of the options our team (support) could assist with, and devs confirmed that none of them are going to be viable options to to bill for this add-on:  It's looking like any solution, will need some level of dev work to set up properly.
The one they did suggest, was looking at billing this as an app or through a specific channel billing. Kevin will be exploring both with their team. 

**Plus:**
If Shopify is adjusting price, there can be impacts on Plus contracts and the Plus team will need to be consulted. Andrea deVries is the plus contact  and Kevin will touch base. 

**Finance:**
There would also need to be finance approval and understanding revenue streams and recognition situations. Billing provided Jennifer Beckert as a contact for Frenzy to reach out to. 

**TLDR  -** there doesn't look to be a quick solution or process for this, and my guess is a larger project would need to be considered by their team. 

If he gets stuck again or needs help/context, indicated he can always reach out. With that nothing further we can help with, and closing this issue out.
